### PR TITLE
v85.10.0

### PR DESCRIPTION
**Pull request by romoore**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,463 @@
+# 85.10.0
+
+This minor release expands support for trait collection and Material Elevation
+changes in several components. Bottom Sheet now supports shadows and elevation.
```
Comment (codeman7) [inline CHANGELOG.md:4]: If we are going to mention bottom sheet's additional support should we also add NavDrawer's support?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,463 @@
...
+* [Store child VCs in `childViewControllers`. (#7992)](https://github.com/material-components/material-components-ios/commit/9ed52651c4390722d1a6b3024e896e1d3008520a) (Robert Moore)
+* [Test for `mdc_overrideBaseElevation` (#8075)](https://github.com/material-components/material-components-ios/commit/bce0f4937b2c1956afef21cbacd434aef5ab6190) (Robert Moore)
+
+### BottomSheet
+
+* [Add elevation property (#8089)](https://github.com/material-components/material-components-ios/commit/43dddb988e341b88b033419d731e09e2cf003de1) (Cody Weaver)
+
+### Buttons
+
+* [Move MDCButton's defaultContentInset configuration to later in init (#8069)](https://github.com/material-components/mater
```
Comment (codeman7) [inline CHANGELOG.md]: I believe this only touches BottomDrawer and catalog scheme, do we traditionally put that in multi component?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,463 @@
...
+
+### List
+
+#### MDCBaseCell
+
+*modified* class: `MDCBaseCell`
+
+| Type of change: | Declaration |
+|---|---|
+| From: | `@interface MDCBaseCell : UICollectionViewCell  /**  The current elevation of the cell.  */ @property(nonatomic, assign) MDCShadowElevation elevation;  /**  By setting this property to @c YES, the Ripple component will be used instead of Ink  to display visual feedback to the user.   @note This property will eventually be enabled by default, deprecated, and then deleted as part  of our migration to Ripple. Learn more at  https://github.com/material-components/material-components-ios/tree/develop/components/Ink#migration-guide-ink-to-ripple   Defaults to NO.  */ @property(nonatomic, assign) BOOL enableRippleBehavior;  /**  The color of the cell’s underlying Ripple.  */ @property(nonatomic, strong, nullable) UIColor *rippleColor;  /**  A block that is invoked when the @c MDCBaseCell receives a call to @c  traitCollectionDidChange:. The block is called after the call to the superclass.  */ @property(nonatomic, copy, nullable) void (^traitCollectionDidChangeBlock)     (MDCBaseCell *_Nonnull cell, UITraitCollection *_Nullable previousTraitCollection);  @end` |
```
Comment (codeman7) [inline CHANGELOG.md]: Should these be like this?

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,463 @@
...
... MDCFeatureHighlightView : UIView  @property(nonatomic, strong, nullable) UIColor *innerHighlightColor UI_APPEARANCE_SELECTOR; @property(nonatomic, strong, nullable) UIColor *outerHighlightColor UI_APPEARANCE_SELECTOR;  @property(nonatomic, strong, nullable) UIFont *titleFont UI_APPEARANCE_SELECTOR; @property(nonatomic, strong, nullable) UIColor *titleColor UI_APPEARANCE_SELECTOR;  @property(nonatomic, strong, nullable) UIFont *bodyFont UI_APPEARANCE_SELECTOR; @property(nonatomic, strong, nullable) UIColor *bodyColor UI_APPEARANCE_SELECTOR;  /*  Indicates whether the view's contents should automatically update their font when the device’s  UIContentSizeCategory changes.   This property is modeled after the adjustsFontForContentSizeCategory property in the  UIContentSizeCategoryAdjusting protocol added by Apple in iOS 10.0.   Default value is NO.  */ @property(nonatomic, readwrite, setter=mdc_setAdjustsFontForContentSizeCategory:)     BOOL mdc_adjustsFontForContentSizeCategory UI_APPEARANCE_SELECTOR;  /**  Enable legacy font scaling curves for Dynamic Type  Default value is NO.  */ @property(nonatomic, readwrite, setter=mdc_setLegacyFontScaling:) BOOL mdc_legacyFontScaling;  @end` |
```
Comment (codeman7) [inline CHANGELOG.md:164]: This seems wrong???

**Comment by romoore** [comment]
Commits all look good!

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,463 @@
...
...te;  /*  Sets the shadow color for a particular control state.   @param elevation The shadow color.  @param state The control state.  */ - (void)setShadowColor:(nullable UIColor *)shadowColor               forState:(UIControlState)state UI_APPEARANCE_SELECTOR;  /*  Returns the title color for a particular control state.   If no title color has been set for a given state, the returned value will fall back to the value  set for UIControlStateNormal.   @param state The control state.  @return The title color for the requested state.  */ - (nullable UIColor *)titleColorForState:(UIControlState)state;  /*  Sets the title color for a particular control state.   @param titleColor The title color.  @param state The control state.  */ - (void)setTitleColor:(nullable UIColor *)titleColor              forState:(UIControlState)state UI_APPEARANCE_SELECTOR;  /**  A block that is invoked when the MDCChipView receives a call to @c  traitCollectionDidChange:. The block is called after the call to the superclass.  */ @property(nonatomic, copy, nullable) void (^traitCollectionDidChangeBlock)     (MDCChipView *_Nonnull activityIndicator, UITraitCollection *_Nullable previousTraitCollection);  @end` |
```
Comment (codeman7) [inline CHANGELOG.md:131]: Same as below

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,3 +1,463 @@
...
... tint color has been set for a state, it will check the value of UIControlStateNormal.  If that value also isn't set, then nil will be returned.   @param state MDCCardCellState the card state  @return The image tint color for the requested state.  */ - (nullable UIColor *)imageTintColorForState:(MDCCardCellState)state UI_APPEARANCE_SELECTOR;  /**  Sets the image tint color for an MDCCardCellStateNormal state   @note The image is only displayed when `selectable` is YES.  @param imageTintColor The image tint color  @param state MDCCardCellState the card state  */ - (void)setImageTintColor:(nullable UIColor *)imageTintColor                  forState:(MDCCardCellState)state UI_APPEARANCE_SELECTOR;  /**  The state of the card cell.  Default is MDCCardCellStateNormal.  */ @property(nonatomic, readonly) MDCCardCellState state;  /**  A block that is invoked when the @c MDCCardCollectionCell receives a call to @c  traitCollectionDidChange:. The block is called after the call to the superclass.  */ @property(nonatomic, copy, nullable) void (^traitCollectionDidChangeBlock)     (MDCCardCollectionCell *_Nonnull collectionCell,      UITraitCollection *_Nullable previousTraitCollection);  @end` |
```
Comment (codeman7) [inline CHANGELOG.md:120]: Same as below

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,3 +1,463 @@
...
...olor for an UIControlState state   If no color has been set for a state, the value for MDCCardViewStateNormal will be returned.  Default value for UIControlStateNormal is blackColor   @param state UIControlState the card state  @return The shadow color for the requested state.  */ - (nullable UIColor *)shadowColorForState:(UIControlState)state UI_APPEARANCE_SELECTOR;  /**  A block that is invoked when the @c MDCCard receives a call to @c  traitCollectionDidChange:. The block is called after the call to the superclass.  */ @property(nonatomic, copy, nullable) void (^traitCollectionDidChangeBlock)     (MDCCard *_Nonnull card, UITraitCollection *_Nullable previousTraitCollection);  /*  The shape generator used to define the card's shape.  When set, layer properties such as cornerRadius and other layer properties are nullified/zeroed.  If a layer property is explicitly set after the shapeGenerator has been set, it will lead to  unexpected behavior.   When the shapeGenerator is nil, MDCCard will use the default underlying layer with  its default settings.   Default value for shapeGenerator is nil.  */ @property(nullable, nonatomic, strong) id<MDCShapeGenerating> shapeGenerator;  @end` |
```
Comment (codeman7) [inline CHANGELOG.md:111]: Same as below

File: CHANGELOG.md (hunk 4 above)
Comment (romoore) [reply CHANGELOG.md:164]: I have no idea how apidiff works. But it always seems wrong to me.

File: CHANGELOG.md (hunk 2 above)
Comment (romoore) [reply CHANGELOG.md]: Since it also modified ShadowElevations (a new elevation constant was added), it seemed accurate.

File: CHANGELOG.md (hunk 1 above)
Comment (romoore) [reply CHANGELOG.md:4]: Done.

File: CHANGELOG.md (hunk 3 above)
Comment (romoore) [reply CHANGELOG.md]: Removed all of this clutter.

**Review by fluxminds — APPROVED** [review]

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,3 +1,187 @@
...
+* [Internal comments to explain font scaling. (#8087)](https://github.com/material-components/material-components-ios/commit/e64c4400356d9eb608add0dd3f7ad7b18322b256) (Robert Moore)
+
+### private/Color
+
+* [add method for regular color blending. (#8077)](https://github.com/material-components/material-components-ios/commit/997b3b2021670b4adddaee3d0abadccc63931df9) (Wenyu Zhang)
+
+## Multi-component changes
+
+* [Add conformance to MDCElevatable, MDCElevationOverriding (#8094)](https://github.com/material-components/material-components-ios/commit/02a707a5b401c8a0321445a77964f0cb38e69440) (Bryan Oltman)
+* [Adds an elevation and shadow to Bottom Drawer. (#8095)](https://github.com/material-components/material-components-ios/commit/7ded6431437dd364bbb6ca0b7ef16f6b6a981ef3) (Yarden Eitan)
```
Comment (codeman7) [inline CHANGELOG.md:179]: Should this be in multi-component?

**Review by codeman7 — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (codeman7) [reply CHANGELOG.md]: I meant the nav-drawer one, just below this one, I think the bottom-app-bar one should be there but I don't think https://github.com/material-components/material-components-ios/commit/7ded6431437dd364bbb6ca0b7ef16f6b6a981ef3 should

File: CHANGELOG.md (hunk 8 above)
Comment (romoore) [reply CHANGELOG.md:179]: Yeah, I think so. It updated the ShadowElevations source.

**Pull request merged**